Index: agente.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import genera_matriz as ma\n\nmatriz = ma.cargar_matriz('matriz_aleatoria.txt')\nfil = matriz.shape[0]\ncol = matriz.shape[1]\n\nparams = {}             #Se crea el diccionario de parametros\n\nfor x in range(0, fil):\n    for y in range(0, col):\n        params[(x, y)] = (('V', False), ('O', False),   ##Se crea una lista de tuplas que definen los\n                          ('I', False), ('X', False))   ##parámetros de cada casilla y se asignan al\n        print(params[(x,y)])                                                ##diccionario según sus coordenadas\n\n\nhumano={0:False,1:1,2:2,3:3,4:4,5:5,6:5}\n\nprint(matriz[0][0])\n\ndef agente():\n\n\n    return param
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/agente.py b/agente.py
--- a/agente.py	(revision 43f445654c0f8620ed312fece61d3918481b9903)
+++ b/agente.py	(date 1619067389279)
@@ -4,20 +4,41 @@
 fil = matriz.shape[0]
 col = matriz.shape[1]
 
-params = {}             #Se crea el diccionario de parametros
+params = {}  # Se crea el diccionario de parametros
+paramsd = {}  # Se crea el diccionario de parametros
 
 for x in range(0, fil):
     for y in range(0, col):
         params[(x, y)] = (('V', False), ('O', False),   ##Se crea una lista de tuplas que definen los
-                          ('I', False), ('X', False))   ##parámetros de cada casilla y se asignan al
-        print(params[(x,y)])                                                ##diccionario según sus coordenadas
+                          ('I', False), ('X', False), ('S',False))
+
+params[(0, 5)] = (('V', False), ('O', False),   ##Se crea una lista de tuplas que definen los
+                          ('I', True), ('X', False), ('S',False))
 
+for x in range(0, fil):
+    for y in range(0, col):
+        paramsd[(x, y)] = {'V': False, 'O': False, 'I': False, 'X': False, 'S': False}
+
+paramsd[(0, 5)] = {'V': True, 'O': False, 'I': True, 'X': False, 'S': False}
 
-humano={0:False,1:1,2:2,3:3,4:4,5:5,6:5}
+humano = {0: False, 1: 1, 2: 2, 3: 3, 4: 4, 5: 5, 6: 5}
 
-print(matriz[0][0])
+
 
 def agente():
+    for i in range(0,fil):
+        for j in range(0,col):
+            if paramsd[(i, j)]['I']:
+                print(paramsd[(i,j)])
+                print(str(i), str(j))
 
+#agente()
 
-    return param
\ No newline at end of file
+for i in range(0, fil):
+    for j in range(0, col):
+        lista_params = params[(i, j)]
+        for param in lista_params:
+            if(param[0]=='I'):
+               if(param[1]):
+                print(param)
+                print(i,j)
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1619048847944)
+++ b/.idea/modules.xml	(date 1619048847944)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/Practica_IA_1.iml" filepath="$PROJECT_DIR$/.idea/Practica_IA_1.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: .idea/Practica_IA_1.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/Practica_IA_1.iml b/.idea/Practica_IA_1.iml
new file mode 100644
--- /dev/null	(date 1619048847904)
+++ b/.idea/Practica_IA_1.iml	(date 1619048847904)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1619048848228)
+++ b/.idea/vcs.xml	(date 1619048848228)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/inspectionProfiles/profiles_settings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/inspectionProfiles/profiles_settings.xml b/.idea/inspectionProfiles/profiles_settings.xml
new file mode 100644
--- /dev/null	(date 1619048848188)
+++ b/.idea/inspectionProfiles/profiles_settings.xml	(date 1619048848188)
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="USE_PROJECT_PROFILE" value="false" />
+    <version value="1.0" />
+  </settings>
+</component>
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1619048790323)
+++ b/.idea/.gitignore	(date 1619048790323)
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1619048848108)
+++ b/.idea/misc.xml	(date 1619048848108)
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.8" project-jdk-type="Python SDK" />
+</project>
\ No newline at end of file
